packages/bot-web-ui/src/components/contract-result-overlay/contract-result-overlay.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
are this eslint comments needed

Reply:
@vinu-deriv In 1 of the case it is required. Will remove the other one